### PR TITLE
api_modules: implement meetup.py, sessionize.py

### PR DESCRIPTION
**Pull request by bogdansharp**
meetup.py Uses publicly accessible (No authorization needed) Meetup GraphQL API to get information about Events. Output is csv file with events.
sessionize.py Uses publicly accessible (No authorization needed) Sessionize Embedding API to get information about Events. Needs sessionize event Id to get data. Output is 2 csv files: Speakers and Sessions for each event.

### CODE REVIEW

File: api_modules/sessionize.py [hunk 1]
```diff
@@ -1,5 +1,122 @@
...
+                    tagline = text_or_default(row.find('h4', class_='sz-speaker__tagline'), "")
+                    bio = text_or_default(row.find('p', class_='sz-speaker__bio'), "")
+                    bio = bio.replace('<br>', '\n')
+                    photo = ''
+                    photo_el = row.find('div', class_='sz-speaker__photo')
+                    if photo_el:
+                        photo_img_el = photo_el.find('img')
+                        if photo_img_el:
+                            photo = photo_img_el.get('src', '')
+                    rows.append([id, name, tagline, bio, photo])
```
Comment (miguelgrinberg) [inline api_modules/sessionize.py:67]: Some of the string fields here appear with leading and/or trailing spaces in the exported CSV. Probably good to call `strip()` on these strings here to remove them.

File: api_modules/sessionize.py [hunk 2]
```diff
@@ -1,5 +1,122 @@
...
+                                end_at = time_arr[3]
+                    speaker_id, speaker = '', ''
+                    speakers_el = row.find('ul', class_='sz-session__speakers')
+                    if speakers_el:
+                        for speaker_el in speakers_el.find_all('li'):
+                            speaker_id += ', ' if len(speaker_id) else ''
+                            speaker_id += speaker_el.get('data-speakerid', '')
+                            speaker += ', ' if len(speaker) else ''
+                            speaker += text_or_default(speaker_el.find('a'), "")
+                    rows.append([id, speaker_id, speaker, title, room_id, room, start_at, end_at, description])
```
Comment (miguelgrinberg) [inline api_modules/sessionize.py:102]: Also strip all strings here.

File: api_modules/sessionize.py [hunk 3]
```diff
@@ -1,5 +1,122 @@
...
+import requests
+import csv
+from bs4 import BeautifulSoup
+
+class Sessionize:
+    def __init__(self, events=None) -> None:
+        default_events = { \
+            'amtv2kwb': 'PyCon 2022', \
+            'jb4vxosa': 'PyCon Limerick 2023', \
+            'jbshwhme': 'PyCon 2023', \
```
Comment (miguelgrinberg) [inline api_modules/sessionize.py:30]: How do you find out the ID of a given event? Also is it possible to get the event name from Sessionize once we know the ID?

File: api_modules/sessionize.py [hunk 4]
```diff
@@ -1,5 +1,122 @@
...
+                        if time_str:
+                            time_arr = time_str.split('|')
+                            if len(time_arr) >= 3:
+                                start_at = time_arr[2]
+                            if len(time_arr) >= 4:
+                                end_at = time_arr[3]
+                    speaker_id, speaker = '', ''
+                    speakers_el = row.find('ul', class_='sz-session__speakers')
+                    if speakers_el:
+                        for speaker_el in speakers_el.find_all('li'):
```
Comment (miguelgrinberg) [inline api_modules/sessionize.py:97]: Putting multiple speakers in a single comma-separated field might make it more complicated to import the data somewhere else. An alternative that is more common in CSV files is to add `speaker1`, `speaker2`, `speaker3`, etc. fields (as many as we think are needed) and then have the speaker entered in separate columns.

Maybe this is okay for now, and you can then change the format if needed.

**Comment by miguelgrinberg** [comment]
This looks pretty good, actually. Great job @bogdansharp!

I've put some minor comments inline, but I got both exporters to work without any issues and without having to change anything in the code. :+1:

File: api_modules/sessionize.py (hunk 1 above)
Comment (bogdansharp) [reply api_modules/sessionize.py:67]: You are right. I am going to add strip()

File: api_modules/sessionize.py (hunk 2 above)
Comment (bogdansharp) [reply api_modules/sessionize.py:102]: Yes

File: api_modules/sessionize.py (hunk 3 above)
Comment (bogdansharp) [reply api_modules/sessionize.py:30]: I take them from similar Sessionize API requests at https://python.ie. I have read the documentation and did not find a way to retrieve the event name by ID. Alongside the Sessionize Embedding API, there exists the regular Sessionize API. However, the available data is the same: schedule, speakers, sessions. Fields specific to the event itself are not available.

File: api_modules/sessionize.py (hunk 4 above)
Comment (bogdansharp) [reply api_modules/sessionize.py:97]: I am going to improve this also.

**Review by miguelgrinberg — APPROVED** [review]
LGTM!

**Pull request merged**